### PR TITLE
TEST: Consult items that match selected topics (see #54)

### PR DESCRIPTION
**Pull request by davp01x**
## Content

Add the scenario and the tests implementations to consult a specific item

---

## Checklist

Please check that your pull request is correct:

- Each commit:
    - [X] corresponds to a contribution that should be notified to users,
    - [X] does not generate new errors or warnings at compile or test time,
    - [X] must be attributed to its real authors (with correct GitHub IDs and [correct syntax for multiple authors](https://help.github.com/articles/creating-a-commit-with-multiple-authors/)).
- The title of a commit should:
    - [X] begin with a contribution type
        - `FEATURE` for a behaviour allowing a user to do something new,
        - `FIX` for a behaviour which has been changed in order to meet user’s expectations,
        - `TEST` when it concerns an acceptance test,
        - `PROCESS` for a change in the way the software is built, tested, deployed,
        - `DOC` when it concerns only internal documentation (however it is better to combine it with the contribution that required this documentation change),
    - [X] be followed by a colon (`:`) with one space after and no space before,
    - [X] be followed by a title as short, as user-centered and as explicit as possible
        - If it is a feature, the title must be the user action (beginning with a verb, and please not `manage`),
        - If it is a fix, the title must describe the intended behavior (with `should`).
    - [X] ends with a reference to the corresponding ticket with the following syntax:
        - `(closes #xx)` if xx is a feature ticket (and the commit is a complete implementation),
        - `(fixes #xx)` if xx is a fix ticket (and the commit is a complete fix),
        - `(see #xx)` otherwise,
- Each committed line is:
    - [X] useful (it would not work if removed)
        - if it is a comment line, its information could not be conveyed by better variables and function naming, better code structuring, or better commit message,
    - [X] related to this very contribution (feature, fix...),
    - [X] in English (with the exception of Gherkin scenarios in French and resulting steps),
    - [X] without any typo in variable, class or function names,
    - [X] correctly indented (spaces rather than tabs, same number of characters as in the rest of the file).

### CODE REVIEW

File: features/step_definitions/item.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Soit("avec comme thème {string}") do |theme|
+
+end
+
+
+# Events
+
+Quand("un visiteur accède à la page d'accueil et clique sur le vitrail {string}") do |item|
+  visit "/"
+  find(:xpath, '//*[@id="root"]/div/div[2]/div[2]/div/div/div[162]/a/img').click
```
Comment (benel) [inline features/step_definitions/item.rb:31]: Test fort compliqué qui risque, à la moindre modification des données ou du code, de plus de ne plus fonctionner.

Conseil : quand vous voyez qu'une interface est difficilement testable, c'est souvent le signe qu'il faut la rendre plus accessible (notamment pour les non-voyants).

File: features/consult_item.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  Et apparaît l attribut du document "spatial Église Saint-Pierre, Dosnon"
+  Et apparaît l attribut du document "name DSN 006 a"
+  Et apparaît l attribut du document "created 2016-07-02"
+  Et apparaît le point de vue "Histoire des religions"
+  Et apparaît le point de vue "Histoire de l'art"
+  Et apparaît le thème "Naissance de Marie"
+  Et apparaît le thème "Anne (mère de Marie)"
+  Et apparaît le thème "Marie (mère de Jésus)"
+  Et apparaît le thème "1862"
+  Et apparaît le thème "Ateliers du Carmel du Mans"
```
Comment (benel) [inline features/consult_item.feature:31]: Intéressant, mais ça ne correspond pas exactement à la fonctionnalité indiquée.

**Comment by benel** [comment]
La fonctionnalité 54 a été testée par un test (f36423e) portant sur une autre fonctionnalité.

**Comment by benel** [comment]
Attention à votre historique. Vous ne respectez pas du tout les règles données ci-dessus.

**Comment by benel** [comment]
Votre vrai nom commence bien par `P` ?

**Comment by davp01x** [comment]
Effectivement, mon nom commence bien par P (Phan). Merci pour les remarques pertinentes effectuées précédemment. Je m'intéresse au plus vite à ceux-ci.

**Pull request closed without merging**